packages/babel-plugin/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
For the time being, I would leave the existing tsconfig.json files as they are targeting **test** directories and not really the source files. Keeping them together with type tests might actually be a requirement of DTslint or something - but this has to be investigated later.

As to the project itself - I think a single `tsconfig.json` in the root will work great for us. This is how TS is set up in Changesets repo.

Reply:
Good point. I'll put those back!